README.rst

```diff
@@ -1,132 +1,200 @@
+  `Sublime Text <https://github.com/coala-analyzer/coala-sublime>`__,
+  `Atom <https://github.com/coala-analyzer/coala-atom>`__,
+  `Vim <https://github.com/coala-analyzer/coala-vim>`__ and
+  `Emacs <https://github.com/coala-analyzer/coala-vim>`__.
+* Optimized performace with multi-threading to parallelize the routines - can
+  complete a 26000 line python repository in less than 3 seconds.
+* File caching support - run only on changed files (experimental).
+
+-----
+
+============
+Installation
+============
+
+To install the **latest stable version** run:
+
+.. code-block:: bash
+
+    $ pip3 install coala-bears
 
-                                                         .o88Oo._
-                                                        d8P         .ooOO8bo._
-                                                        88                  '*Y8bo.
-                                          __            YA                      '*Y8b   __
-                                        ,dPYb,           YA                        68o68**8Oo.
-                                        IP'`Yb            "8D                       *"'    "Y8o
-                                        I8  8I             Y8     'YB                       .8D
-                                        I8  8P             '8               d8'             8D
-                                        I8  8'              8       d8888b          d      AY
-         ,gggo,    ,ggggo,    ,gggo,gg  I8 dP    ,gggo,gg   Y,     d888888         d'  _.oP"
-        dP"  "Yb  dP"  "Y8go*8P"  "Y8I  I8dP    dP"  "Y8I    q.    Y8888P'        d8
-       i8'       i8'    ,8P i8'    ,8I  I8P    i8'    ,8I     "q.  `Y88P'       d8"
-      ,d8,_    _,d8,   ,d8' d8,   ,d8b,,d8b,_ ,d8,   ,d8b,       Y           ,o8P
-    ooP""Y8888PP*"Y8888P"   "Y8888P"`Y88P'"Y88P"Y8888P"`Y8            oooo888P"
+|Stable|
 
-.. Stop ignoring LineLengthBear
+To install the latest development version run:
 
-Get *coala* to lint all your languages in your project with one tool and
-config!
+.. code-block:: bash
 
-Demo (Click to View)
---------------------
+    $ pip3 install coala-bears --pre
 
-|asciicast|
+The latest code from the master branch is automatically deployed as the
+development version in PyPI.
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
-   :target: https://asciinema.org/a/42968?autoplay=1
-   :width: 100%
+|PyPI| |Windows| |Linux|
 
-About
 -----
 
-*coala* is a language independent analysis toolkit. It empowers developers
-to create rules which a project's code should conform to. *coala* takes care
-of showing these issues to users in a friendly manner, is versatile and can be
-used in any environment. Patches to automatically fix code will be managed too.
-*coala* has a set of official bears (plugins) to provide an out-of-the-box
-analysis functionality for many popular languages in addition to some
-generically applicable algorithms.
+=====
+Usage
```

Comment:
this section requires the most care imo, im sorry for making you go over this again but i think you could modify it a little.
Try making it a step-based guide.